projects/documentation/scripts/copy-component-docs.js

```diff
@@ -233,6 +300,7 @@ async function main() {
         `${projectDir}/(packages|tools)/**/*.md`
     )) {
         await processREADME(mdPath);
+        await processChangelog(mdPath);
```

Comment:
Why this needs to be synchronous? Do we need to wait for processREADME to complete?

Reply:
no we don't. i've made it synchronous now.